components/faction/commons/faction.lua

```diff
@@ -104,11 +116,13 @@ function Faction.read(faction, options)
 		return nil
 	end
 
-	options = options or {}
+	if type(options) ~= 'table' then
```

Comment:
am on mobile comment is referring to this plus the next 2 lines

could make it 1 line
`options = type(options) == 'table' and options or {}`